wiki/versions/1.19/entity_attributes/markdown/entity_attributes.md

```diff
@@ -0,0 +1,148 @@
+
+At this point, we might want to add a language entry:
+
+```json
+{
+  "attribute.name.generic_jump_boost": "Jump Boost"
+}
+```
+
+With our `EntityAttribute` instance created, we can go ahead and register it.
+
+```file:src/main/java/org/quiltmc/wiki/entity_attributes/AttributesExample.java@Register-Attribute
+```
+
+This identifier doesn't follow normal conventions, but it's what the vanilla attributes use.
+
+## Registering Attributes on Entities
+
+The next step is letting certain entities know that they accept our new attribute. Let's walk through how entities do this in the first place.
+
+### How Entities Know Their Attributes
+
+- Each entity class has some sort of static attribute creation method returning a `DefaultAttributeContainer.Builder`. Here's what the one in `LivingEntity` looks like:
+
+```java
+public static DefaultAttributeContainer.Builder createLivingAttributes() {
+    return DefaultAttributeContainer.builder()
+        .add(EntityAttributes.GENERIC_MAX_HEALTH)
+        .add(EntityAttributes.GENERIC_KNOCKBACK_RESISTANCE)
+        .add(EntityAttributes.GENERIC_MOVEMENT_SPEED)
+        .add(EntityAttributes.GENERIC_ARMOR)
+        .add(EntityAttributes.GENERIC_ARMOR_TOUGHNESS);
+}
+```
+
+This builder is still mutable at this point, so other entities call `LivingEntity.createLivingAttributes` and build off of it when defining their own accepted attributes.
+
+- This method gets called in the `DefaultAttributeRegistry` class when creating its namesake map (`DEFAULT_ATTRIBUTE_REGISTRY`). This maps entity types to their corresponding `DefaultAttributeContainers`, now built and immutable. This map is immutable *and* private, but the class has wrappers for `get` and `containsKey`.
+
+> When creating your own entity classes, you can use `FabricDefaultAttributeRegistry.register` to take care of this without your own mixins.
+
+- During play, the `LivingEntity` constructor initializes the `attributes` field as a new `AttributeContainer` instance based on the result of the getter in `DefaultAttributeRegistry` mentioned prior. This is the value returned in the `LivingEntity#getAttributes` getter.
+
+- When *using* an attribute's value stored in an entity, you need to retrieve its corresponding `EntityAttributeInstance`; we'll go over this in more detail later. For now, all you should know is that **if the entity doesn't have an attribute in its `attributes` field, it won't have that attribute instance**, and you won't be able to apply a modifier.
+
+### Mixing Into Attribute Creation
+
+Each entity that does something different with attributes has their own attribute creation method, which you can find in `DefaultAttributeRegistry`. Since we want our attribute on all entities, we'll mix into the `LivingEntity.createLivingAttributes` method shown in the previous section.
+
+Remember, at this point, the builder is still immutable - and a reference - so our mixin is very straightforward.
```

Comment:
Shouldn't it be "the builder is still mutable"?

Reply:
Yep, my mistake